partner_contact_department/__openerp__.py

```diff
@@ -11,7 +11,7 @@
     "summary": "Assign contacts to departments",
     'version': '9.0.1.0.0',
     'category': 'Customer Relationship Management',
-    'author': 'Tecnativa S.L. - Vicent Cubells, '
+    'author': 'Tecnativa - Vicent Cubells, '
```

Comment:
Sorry, I wasn't see the second part, but it's only Tecnativa at all, also without "Vicent Cubells". Your name is put on the README in the contributors section and also on each file header.